project.clj

```diff
@@ -1,33 +1,43 @@
-(defproject one-time "0.7.0"
+(defproject one-time "0.7.1"
```

Comment:
can you drop this change, i'll do the version bumps separately. Everything looks good otherwise, I'll test it later tonight and (if no issues) will merge it.

Reply:
Ofcourse: 9e4363e307d771b8be0546189faddb286c6dceec